content/blog/azure-v6-release/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Pulumi Azure Classic Provider Version 6.0.0"
+date: 2024-09-24T15:04:00
+draft: false
+meta_desc: "Release of the next version of the Azure Classic Pulumi Provider"
+meta_image: meta.png
+authors:
+    - daniel-bradley
+tags:
+    - azure
+    - release
+---
+
+The latest major release of the Azure Classic Pulumi Provider is available now!
+Our 6.0 release contains the [latest upstream changes](https://github.com/hashicorp/terraform-provider-azurerm/releases/tag/v4.0.1) to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Azure Classic provider can be used to provision any of the Azure resources available in the upstream provider.
```

Comment:
Maybe add a sentence to disambiguate between the two providers?

"While Pulumi's Azure Native provider is generally recommended for Azure, the Azure Classic provider that's based on Terraform can be a useful alternative."

Reply:
Yeah - definitely worth a note on this. Perhaps even a note block that stands out from the main body text. Worth making clear this is not a provider most Pulumi Azure users should be using by default early in this post.

Reply:
Done 👍